source/documentors/references/doc_english_writing.rst

```diff
@@ -40,9 +34,7 @@ However!
 
 Example:
 
-"Block open port on catheter fitting" could mean completely different things
-once the definite article is inserted. Compare "Block the open port on the
-catheter fitting." to "Block open the port on the catheter fitting.
+"Block open port on catheter fitting" could mean completely different things once the definite article is inserted. Compare "Block the open port on the catheter fitting." to "Block open the port on the catheter fitting.
```

Comment:
This is kind of a silly example and may be confusing in the Open edX context, because I wouldn't expect everyone to know the medical concepts of a "catheter" or a "port". But I can't think of something better. Can you?